chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -147,7 +147,7 @@ def boolean_input(id_: str, required: bool = True):
         wm.WorkflowFileArrayInput(
             id="document_files",
             required=True,
-            pattern=r"^.*\.(pdf|csv|tsv|txt|docx|xlsx|jpeg|jpg|png|gif|md|markdown|mp3|m4a|mp4)$",
+            pattern=r"^.*\.(pdf|csv|tsv|txt|docx|xlsx|jpeg|jpg|png|gif|md|html|markdown|mp3|m4a|mp4)$",
```

Comment:
put html before or after the two markdown (md, markdown) entries instead of splitting them up